src/View/Helper/BakeHelper.php

```diff
@@ -78,7 +78,19 @@ public function stringifyList(array $list, array $options = [])
                 $v = "'$v'";
             }
             if (!is_numeric($k)) {
-                $v = "'$k' => $v";
+                $nestedOptions = $options;
+                if ($nestedOptions['indent']) {
+                    $nestedOptions['indent'] += 1;
+                }
+                if (is_array($v)) {
+                    $v = sprintf(
+                        "'%s' => [%s]",
+                        $k,
```

Comment:
Do we need to escape possible `'` in `$k`?

Reply:
I don't see anyone using single quotes in config key names.

Reply:
Ok.